src/main/kotlin/no/nav/familie/ef/sak/brev/FrittståendeBrevService.kt

```diff
@@ -44,15 +46,16 @@ class FrittståendeBrevService(
                 brevtype = utledFrittståendeBrevtype(frittståendeBrevDto.brevType),
                 fil = brev,
                 journalførendeEnhet = journalførendeEnhet,
-                saksbehandlerIdent = SikkerhetContext.hentSaksbehandler(true),
+                saksbehandlerIdent = saksbehandlerIdent,
                 mottakere = mapMottakere(frittståendeBrevDto)
             )
         )
+        mellomlagringBrevService.slettMellomlagretFrittståendeBrev(fagsak.id, saksbehandlerIdent)
     }
 
     private fun mapMottakere(frittståendeBrevDto: FrittståendeBrevDto): List<Brevmottaker>? {
         val mottakere = frittståendeBrevDto.mottakere
-        return if(mottakere == null || (mottakere.personer.isEmpty() && mottakere.organisasjoner.isEmpty())) {
+        return if (mottakere == null || (mottakere.personer.isEmpty() && mottakere.organisasjoner.isEmpty())) {
```

Comment:
Ikke kommentar til denne pr'en, men trenger denne returnere null?

Reply:
Ja vi "trengte" det for å kunne håndtere gamle og nye versjoner av frontend/iverksett sånn att det virket selv når vi ikke hadde oppdatert allt annet
Blir slettet i https://github.com/navikt/familie-ef-sak/pull/1816/files#diff-2c80d07ccf7d3a14b831900bed5f0363d383705b88a194bacc67dd23fff9867bR55 når frontend er oppdatert